src/app/icon/almicon.directive.ts

```diff
@@ -39,11 +39,15 @@ export class AlmIconDirective implements OnInit, OnChanges {
     });
 
     if (this.iconType in IconMap) {
-      IconMap[this.iconType].forEach((item: any) => {
+      this.iconColor = IconMap[this.iconType].colour;
```

Comment:
We are using the american spelling of 'colour' everywhere. To keep it consistent let's use 'color' instead of 'colour'